### PR TITLE
Release 0.11.0

### PR DESCRIPTION
**Pull request by nearmax**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# 0.11.0
+## API breaking changes
+
+* Renamed `Map` to `UnorderedMap` and `Set` to `UnorderedSet` to reflect that one cannot rely on the order of the elements in them. In this PR and in https://github.com/near/near-sdk-rs/pull/154
+
+## Non-API breaking changes
```
Comment (nearmax) [inline CHANGELOG.md:6]:
```suggestion
## Non-API-breaking changes
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# 0.11.0
+## API breaking changes
+
+* Renamed `Map` to `UnorderedMap` and `Set` to `UnorderedSet` to reflect that one cannot rely on the order of the elements in them. In this PR and in https://github.com/near/near-sdk-rs/pull/154
+
+## Non-API-breaking changes
```
Comment (nearmax) [inline CHANGELOG.md:6]:
```suggestion
## Other changes
```

**Comment by nearmax** [comment]
CI seems to be stuck. CC @ailisp Force merging since I ran the checks locally.

**Comment by ailisp** [comment]
@nearmax It seems travis simply stuck at launching a runner vm machine: https://travis-ci.com/github/near/near-sdk-rs/jobs/346341231 should we switch to github actions, which is currently used in nearcore mac release build and seems very reliable? Or buildkite, which is not free (cost is self host instance)

**Comment by nearmax** [comment]
> @nearmax It seems travis simply stuck at launching a runner vm machine: https://travis-ci.com/github/near/near-sdk-rs/jobs/346341231 should we switch to github actions, which is currently used in nearcore mac release build and seems very reliable? Or buildkite, which is not free (cost is self host instance)

I trust you to have more knowledge about it than I do, so let's do whatever you think is better, e.g. more ergonomic. I doubt buildkite is too expensive, so if you think it is ergonomically better then let's go with it.

**Pull request merged**